exe/kubernetes-deploy

```diff
@@ -13,7 +13,7 @@ verbose_log_prefix = false
 
 ARGV.options do |opts|
   opts.on("--bindings=BINDINGS", "Expose additional variables to ERB templates " \
-    "(format: k1=v1,k2=v2 or JSON)") do |binds|
+    "(format: k1=v1,k2=v2, JSON string or JSON file path)") do |binds|
```

Comment:
maybe worth mentioning the `@` prefix here?

Reply:
Added